src/OrchardCore.Modules/OrchardCore.Security/Drivers/SecuritySettingsDisplayDriver.cs

```diff
@@ -77,7 +74,7 @@ public override async Task<IDisplayResult> EditAsync(SecuritySettings settings,
             }).Location("Content:2").OnGroup(SettingsGroupId);
         }
 
-        public override async Task<IDisplayResult> UpdateAsync(SecuritySettings section, UpdateEditorContext context)
+        public override async Task<IDisplayResult> UpdateAsync(SecuritySettings settings, UpdateEditorContext context)
```

Comment:
Revert all of these parameter renames, please. The overridden method uses the name `section`. As it is the general best practice and something that code analysis also checks, we should use the same names (also when implementing interfaces) as in the parent type.

Reply:
every where else we call it settings. I like to keep them all settings for consistency. only very few drivers that show section.

Reply:
Then everywhere else is wrong too, unfortunately. The driver could be as well, but given that it's for a section of a settings model (and called **Section**DisplayDriver), I think that's correct.

Reply:
I disagree. section is a generic term used by the generic base class. The [SecuritySettingsDisplayDriver.cs](https://github.com/OrchardCMS/OrchardCore/pull/15875/files/a183650dea8427014ed7da9b466dc6c2a70eab25#diff-aac30efd116bc5bdb80038fe570cb21ed4d78e70a197ba8437639db13664affd) provides a concrete implementation in this case SecuritySettings. It's okay for the generic implementation to call it section since it is not always settings. But, in the concrete implementation, we know what the section is actually is.

Reply:
Let's agree to disagree.